src/libs/actions/IOU.ts

```diff
@@ -8635,7 +8635,7 @@ function canApproveIOU(
     const isCurrentUserManager = managerID === userAccountID;
     const isOpenExpenseReport = isOpenExpenseReportReportUtils(iouReport);
     const isApproved = isReportApproved({report: iouReport});
-    const iouSettled = isSettled(iouReport?.reportID);
+    const iouSettled = isSettled(iouReport);
     const reportNameValuePairs = chatReportRNVP ?? getReportNameValuePairs(iouReport?.reportID);
```

Comment:
Conflict was here. I changed it to what is currently on staging - https://github.com/Expensify/App/blob/staging/src/libs/actions/IOU.ts#L8639